### PR TITLE
Added cleanup steps for persistent storage

### PR DESCRIPTION
**Pull request by Kartik494**
Fixes: #31114 

This PR added clean up steps for persistent volumes used in the tutorial.

### CODE REVIEW

**Review by chrisnegus — COMMENTED** [review]
/lgtm

File: content/en/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 1]
```diff
@@ -1202,12 +1202,48 @@ Service:
...
+```
+persistentvolumeclaim "www-web-0" deleted
+persistentvolumeclaim "www-web-1" deleted
+persistentvolumeclaim "www-web-2" deleted
+persistentvolumeclaim "www-web-3" deleted
+persistentvolumeclaim "www-web-4" deleted
+```
+
+```shell
+kubectl get pvc
```
Comment (sftim) [inline content/en/docs/tutorials/stateful-application/basic-stateful-set.md:1244]: What about the PVs? Remember that the page says:

> If your cluster is not configured to do so, you will have to manually provision two 1 GiB volumes prior to starting this
tutorial.

at the start

Comment (Kartik494) [reply content/en/docs/tutorials/stateful-application/basic-stateful-set.md:1244]: Hi @sftim Actually PV's will be successfully delete when i delete PVC's so should i explicitly mention output of `PV` status after cleaning up of pvc's i.e 

```
kubectl get pv 
No resources found
 
```

Comment (Kartik494) [reply content/en/docs/tutorials/stateful-application/basic-stateful-set.md:1244]: Hi @sftim Could you please suggest me on  this.

> Hi @sftim Actually PV's will be successfully delete when i delete PVC's so should i explicitly mention output of `PV` status after cleaning up of pvc's i.e
> 
> ```
> kubectl get pv 
> No resources found
>  
> ```

Comment (Kartik494) [reply content/en/docs/tutorials/stateful-application/basic-stateful-set.md:1244]: Hi @sftim could you please suggest on should i document  delete PV's step also?

Comment (annajung) [reply content/en/docs/tutorials/stateful-application/basic-stateful-set.md:1244]: I think just a reminder at the end that if you manually provisioned volumes prior to the tutorial, it needs to be cleaned up manually should be sufficient

Comment (Kartik494) [reply content/en/docs/tutorials/stateful-application/basic-stateful-set.md:1244]: @annajung Sure i will add the reminder at the end . Thanks for suggestions.

Comment (Kartik494) [reply content/en/docs/tutorials/stateful-application/basic-stateful-set.md:1244]: @annajung @sftim  i have kept the note regarding the same. PTAL

**Comment by Sea-n** [comment]
FYI, you can do this to keep trigger retest while keeping <kbd>lgtm</kbd> label.

```
git commit --amend --no-edit
git push -f
```

Let me try another thing, maybe this command will work.
/retest

**Comment by Kartik494** [comment]
@Sea-n Thanks for the information , Jobs are now running.

**Comment by Kartik494** [comment]
@Sea-n @annajung  I have addressed the changes . PTAL

**Comment by sftim** [comment]
Changes since https://github.com/kubernetes/website/pull/35017#pullrequestreview-1039551417 are minor and beneficial

We should still inform users that they might need to delete the PV. We shouldn't 100% assume that the PVs were automatically provisioned:
> This tutorial assumes that your cluster is configured to dynamically provision PersistentVolumes. If your cluster is not configured to do so, you will have to manually provision two 1 GiB volumes prior to starting this tutorial.

We can mention the manual cleanup for PVs at the end of the tutorial, at the end of the note I quoted, or in both places.

Even with that snag outstanding, this feels like an improvement.
/lgtm

File: content/en/docs/tutorials/stateful-application/basic-stateful-set.md (hunk 1 above)
Comment (reylejano) [reply content/en/docs/tutorials/stateful-application/basic-stateful-set.md:1244]: > Hi @sftim Actually PV's will be successfully delete when i delete PVC's so should i explicitly mention output of `PV` status after cleaning up of pvc's i.e
> 
> ```
> kubectl get pv 
> No resources found
>  
> ```

Others have replied about this, PVs will only be automatically deleted if the PVs were dynamically provisioned
PVs that were manually created will have to be deleted manually e.g. `kubectl delete pv pv1`

**Comment by reylejano** [comment]
non-blocking nit: clearer instructions to delete manually created PVs
/approve

**Pull request merged**